.env

```diff
@@ -14,4 +14,7 @@ DOCKER_EXTRA_ARGS=" -f docker-compose.default.yml"
 #DOCKER_BASE_DIR=""
 
 # change the jsdocker-compose directory
-DOCKER_JSDC_DIR="/apps/jsdocker-compose"
\ No newline at end of file
+DOCKER_JSDC_DIR="/apps/jsdocker-compose"
+
+# webserver port
+DOCKER_PORT=81
```

Comment:
Probably you want to comment it out and set the 8080 port instead to reflect the docker-compose default value